components/feature/readerview/src/main/assets/extensions/readerview/moment/moment-2.24.0.js

```diff
@@ -0,0 +1,14765 @@
+;(function (global, factory) {
```

Comment:
We can use the [minified version](https://momentjs.com/downloads/moment.min.js) to save some bits or minify it our self

Reply:
@Amejia481 yes, good idea! For now I want to be able to read / debug the code though. We can land the minified version or swap out in release builds.

Reply:
Filed to investigate how we want to do this for release builds: https://github.com/mozilla-mobile/android-components/issues/2288

Reply:
👍